scripts/closure-compiler/build-devapp-bundle.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Script that bundles the dev-app using the Google Closure compiler.
+# This is script is used to verify closure-compatiblity of all Material components.
+
+set -e -o pipefail
+
+# Go to the project root directory
+cd $(dirname $0)/../..
+
+# Build the demo-app and also create the release output.
+$(npm bin)/gulp build:devapp
+$(npm bin)/gulp :package:release
+
+# Rebuild demo-app with ES2015 modules. Closure compiler is then able to parse imports.
+$(npm bin)/tsc -p src/demo-app/ --target ES2015 --module ES2015
+
+# Re-compile RxJS sources into ES2015. Otherwise closure compiler can't parse it properly.
+$(npm bin)/ngc -p scripts/closure-compiler/tsconfig-rxjs.json
+
+# Create a list of all RxJS source files.
+rxjsSourceFiles=$(find dist/packages/rxjs -name '*.js');
+
+# Due a Closure Compiler issue https://github.com/google/closure-compiler/issues/2247
+# we need to add exports to the different RxJS ES2015 files.
+for i in $rxjsSourceFiles; do
+    echo "export var __CLOSURE_WORKAROUND__" >> $i
+done
+
+OPTS=(
```

Comment:
now that we use a flagfile maybe it would be better to formulate this with 
```
cat > flagfile << EOF
...
EOF
```

instead of a bash array?